<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WidgetJobFile</class>
 <widget class="QWidget" name="WidgetJobFile">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>503</width>
    <height>338</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBoxJobName">
     <property name="title">
      <string>作业名称</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLineEdit" name="lineEditJobName"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBoxFileName">
     <property name="title">
      <string>脚本文件</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLineEdit" name="lineEditFileName"/>
      </item>
      <item>
       <widget class="QToolButton" name="toolButtonSelectFile">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBoxMethodName">
     <property name="title">
      <string>执行过程</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLineEdit" name="lineEditMethodName"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBoxParams">
     <property name="title">
      <string>参数列表（以逗号分割）</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLineEdit" name="lineEditParams"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lineEditJobName</tabstop>
  <tabstop>lineEditFileName</tabstop>
  <tabstop>toolButtonSelectFile</tabstop>
  <tabstop>lineEditMethodName</tabstop>
  <tabstop>lineEditParams</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
